microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/integration/ModuleUtil.java

```diff
@@ -0,0 +1,32 @@
+package org.jboss.eap.qe.microprofile.metrics.integration;
+
+import java.io.File;
+
+import org.apache.commons.io.FileUtils;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.exporter.ZipExporter;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.wildfly.extras.creaper.commands.modules.AddModule;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+public class ModuleUtil {
```

Comment:
@istraka I am sorry - I missed this class before. Could you please add some basic JavaDoc specyfing what is the purpose of this class? I suggest, that based on code, it installs module to WildFly. If that is the case, I suggest we move it to another module (after this will be merged to speed up the process) so it can be used testsuite wide. WDYT?

Reply:
Agree.

Reply:
resolving, expecting to be fixed soon in new PR